### PR TITLE
LPS-188812 EVP Automated Test| master

### PR DESCRIPTION
**Pull request by joelrocha68**
[LPS-188812](https://liferay.atlassian.net/browse/LPS-188812)

### CODE REVIEW

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:test:osb-site-initializer-evp

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 1]
```diff
@@ -61,6 +57,34 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-172949. Verify that is possible finance user can edit the Organization with 'Awaiting Finance Review' status"
```
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:60]: Hey @joelrocha68, what you think about this new Scenario description
`"LPS-172949. Verify that is possible finance user can edit the Organization with 'Awaiting Finance Review' status"`

**Review by renato-araujo-liferay — CHANGES_REQUESTED** [review]
Hey @joelrocha68, what you think about this new Scenario description
"LPS-172949. Verify that is possible finance user can edit the Organization with 'Awaiting Finance Review' status"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase (hunk 1 above)
Comment (renato-araujo-liferay) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:60]: Hey @joelrocha68, what you think about this new Scenario description
"LPS-172949 - Finance user should edit Organizations with 'Awaiting Finance Review' status"

**Review by renato-araujo-liferay — CHANGES_REQUESTED** [review]
"LPS-172949 - Finance user should edit Organizations with 'Awaiting Finance Review' status"

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase [hunk 2]
```diff
@@ -61,6 +57,34 @@ definition {
 		}
 	}
 
+	@description = "LPS-172949 - Finance user should edit Organizations with 'Awaiting Finance Review' status"
+	@priority = 4
+	test FinanceUserCanEditOrganization {
```
Comment (LuanaKSilva) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase:62]: Hey @joelrocha68, the test name must begin with an action. Could you change it please?
Sugestion: `CanFinanceUserEditOrganization`

**Review by LuanaKSilva — CHANGES_REQUESTED** [review]
Hey @joelrocha68, thanks for working on it!
I have requested some adjustments, please take a look.

**Comment by joelrocha68** [comment]
ci:forward

**Comment by joelrocha68** [comment]
ci:test:sf

**Comment by joelrocha68** [comment]
ci:forward

**Pull request closed without merging**